grammar.js

```diff
@@ -1035,6 +1115,10 @@ function commaSep1(rule) {
   return sep1(rule, ",");
 }
 
+function commaSep(rule) {
```

Comment:
I am not sure I'll probably rewrite it as `optional(commaSep1(...))` in the future.